hack/queue_conf.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+"""Set up queue attributes before worker start up so we are sure all queues are properly set up."""
+
+import boto3
+import os
+import logging
+import sys
+from datetime import timedelta
+
+logging.basicConfig(level=logging.WARNING)
+_logger = logging.getLogger(__name__)
+# Make verbose only logger in this script, botocore tends to be super
+# verbose when performing requests
+_logger.setLevel(logging.DEBUG)
+
+
+_AWS_SQS_REGION = os.getenv('AWS_SQS_REGION', 'us-east-1')
+_AWS_ACCESS_KEY_ID = os.getenv('AWS_SQS_ACCESS_KEY_ID')
+_AWS_SECRET_ACCESS_KEY = os.getenv('AWS_SQS_SECRET_ACCESS_KEY')
+
+
+def set_queue_attributes(queue_names):
+    """Set required queue attributes for all queues used by worker.
+
+    :param queue_names: a list of queue names to set attribute to
+    :type queue_names: list
+    """
+    client = boto3.client(
+        'sqs',
+        aws_access_key_id=_AWS_ACCESS_KEY_ID,
+        aws_secret_access_key=_AWS_SECRET_ACCESS_KEY,
+        region_name=_AWS_SQS_REGION
+    )
+
+    for queue_name in queue_names:
+        _logger.info("Creating or requesting QueueURL for queue '{}'".format(queue_name))
+        response = client.create_queue(QueueName=queue_name)
+
+        queue_url = response.get('QueueUrl')
+        if not queue_url:
+            raise RuntimeError("Response from remote does not contain QueueUrl, "
+                               "cannot adjust queue attributes, response: {}".format(response))
+
+        response = client.set_queue_attributes(
+            QueueUrl=queue_url,
+            Attributes={
+                'MessageRetentionPeriod': int(timedelta(days=14).total_seconds())
```

Comment:
Shouldn't it be configurable ?

Reply:
The way how core works, we will need to make sure AWS retains messages as long as possible.